src/search/search.js

```diff
@@ -63,11 +63,12 @@ export const search = new Svue({
     },
     start(prevUrls, results, params) {
       if (results == null) return 0;
-      var startIndex = 1;
-      if (Array.isArray(prevUrls) && params && !isNaN(params.perPage)) {
-        startIndex += prevUrls.length * params.perPage;
-      }
-      return startIndex;
+      return prevUrls.length * params.perPage + 1;
+      //var startIndex = 1;
+      //if (Array.isArray(prevUrls) && params && !isNaN(params.perPage)) {
+      //  startIndex += prevUrls.length * params.perPage;
+      //}
+      //return startIndex;
```

Comment:
What is this change for?

Reply:
It's not a change I made. There was a lot of stuff on staging before I merged my changes in. I'll see if I can figure out where it came from.

Reply:
Here's the commit: https://github.com/MuckRock/documentcloud-frontend/commit/52faf7f24fbd109c0742f7e397f63b98c390a35d Looks like something @jimkang was testing. Should we leave it in or revert?

Reply:
It shouldn't be in. Another thing that I realized: You're merging staging to prod, which we don't really do. I can see how this can be confusing, but what's on staging is usually disposable — lots of force pushes to staging happen.

I think you actually want to make the PR from your feature branch straight to master.

Reply:
OK. I would like to get to a point where we have a deployed staging branch that is identical to master except for the change we're making, but that's a larger thing we should figure out. I'll close this and make a new PR.